api/src/utils/create_entity_utils.py

```diff
@@ -23,7 +23,7 @@ def set_default(attr: BlueprintAttribute, blueprint_provider):
         return attr.default
 
     # Setting default, default values...
-    match attr.attribute_type:
+    match attr.attribute_type:  # noqa E999
```

Comment:
```suggestion
    match attr.attribute_type:
```
Not sure if I remember incorrectly, but shouldn't this be fixed with a version bump to pep8? (@soofstad)

Reply:
upgrading flake8 seems to fix this